packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -32,9 +32,10 @@ type Props = {|
     opened?: boolean,
 
     /**
-     * Called when the menu closes
+     * In controlled mode, use this prop in case the parent needs to be notified
+     * when the menu opens/closes.
      */
-    onClose?: () => mixed,
+    onToggle?: (opened: boolean) => mixed,
```

Comment:
Technically this would be a breaking change, but since we haven't released it yet this is ok!